easybuild/easyblocks/l/lammps.py

```diff
@@ -178,7 +175,24 @@ def translate_lammps_version(version):
        "NOV": '11',
        "DEC": '12'
     }
-    return '.'.join([items[2], month_map[items[1].upper()], '%02d' % int(items[0])])
+    items = [x for x in re.split('(\\d+)', version) if x]
+    if len(items) < 3:
+        raise ValueError("Version %s does not have (at least) 3 elements" % version)
+    try:
+        return '.'.join([items[2], month_map[items[1].upper()], '%02d' % int(items[0])])
+    except KeyError:
+        # avoid failing miserably under --module-only --force
+        if os.path.exists(path) and os.listdir(path):
+            with open(os.path.join(path, 'src', 'version.h')) as file:
+                file_contents = file.read()
+            lines = re.split('\n', file_contents)
```

Comment:
use `read_file` function here:
```suggestion
            txt = read_file(os.path.join(path, 'src', 'version.h'))